### PR TITLE
Fixing Crypto/Defi/DPI Bug

### LINKED ISSUE #2909: [Bug] Crypto/Defi/DPI Columns Error
**Describe the bug**
The `crypto/defi/dpi` command is breaking on the integration test and in the terminal.

Integration Test [(Reference)](https://github.com/OpenBB-finance/OpenBBTerminal/actions/runs/3275267958/jobs/5389893393)
Screen Shot 2022-10-18 at 7 11 04 PM

Terminal
Screen Shot 2022-10-18 at 7 11 32 PM

**To Reproduce**
`crypto/defi/dpi`

### PR DESCRIPTION
**Pull request by simmonsj330**
# Description
Fixes #2909 by adding additional "30D Users" data column. This must've previously not been tracked by Defi Pulse, but now is as seen below.

Screen Shot 2022-10-18 at 7 49 48 PM

As a result, the command now works properly.

Screen Shot 2022-10-18 at 7 49 22 PM

# How has this been tested?
Re-ran Unit Tests and have included them in this PR.

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by jmaslek** [comment]
That screenshot does not appear to be sorted in order of Users_30D

**Comment by simmonsj330** [comment]
@jmaslek Fixed! 

Screen Shot 2022-10-18 at 8 22 08 PM

**Comment by jmaslek** [comment]
So this works, but the ascend is off.  My guess is that the column dtypes are string and not numeric

Screen Shot 2022-10-18 at 8 43 56 PM

**Comment by simmonsj330** [comment]
@jmaslek Made changes to tvl: 
Screen Shot 2022-10-19 at 12 50 06 PM

**Comment by jmaslek** [comment]
I think I was also seeing that for the 1 Day %.  The notation for using "M" is much cleaner.  Not sure is its easier to convert than re-convert, or is sort__values has a key=int equivalent.

File: openbb_terminal/cryptocurrency/defi/defipulse_model.py [hunk 1]
```diff
@@ -43,11 +43,31 @@ def get_defipulse_index(sortby: str = "TVL", ascend: bool = False) -> pd.DataFra
...
+        value = value.replace("$", "")
+        value = value.replace("M", "000000")
+        value = value.replace("B", "000000000")
+        value = value[0:-residue]
+        value = value.replace(".", "")
+        df.at[i, "TVL_$"] = int(value)
 
+        percent = row["1_Day_%"]
+        percent = percent.replace("%", "")
+        percent = percent.replace("+", "")
```
Comment (jmaslek) [inline openbb_terminal/cryptocurrency/defi/defipulse_model.py:66]: For future reference,. there is already a helper_func: `lambda_clean_data_values_to_float` that you can just use to turn the string into a number

**Review by jmaslek — APPROVED** [review]

**Pull request merged**